docs/node/run-your-node/paratime-observer-node.mdx

```diff
@@ -0,0 +1,142 @@
+confidential smart contact queries. If you just want to run a _ParaTime client_ node, see the
+[instructions for running a ParaTime client node](paratime-client-node.mdx). If you want to run a _ParaTime_ node
+instead, see the [instructions for running a ParaTime node](paratime-node.mdx). Similarly, if you want to run a
+_validator_ or a _non-validator_ node instead, see the [instructions for running a validator node](validator-node.mdx)
+or [instructions for running a non-validator node](non-validator-node.mdx).
+
+:::
+
+:::tip
+
+[TEE support] and a ParaTime client node with a confidential ParaTime is required to run a ParaTime observer node.
+
+:::
+
+:::tip
+
+There may be per-ParaTime on-chain policy requirements (such as whitelisting) for running observer nodes.
+
+:::
+
+This guide will cover setting up your ParaTime observer node for the Oasis Network. Observer nodes are ParaTime
+client nodes that support confidential queries without being elected into the compute committee. They are registered on
+chain so that their eligibility can be enforced by an on-chain policy (e.g. key manager committees can grant them
+permissions). This way users can, for example, run confidential transactions and view calls on [Sapphire] ParaTime.
+This guide assumes some basic knowledge on the use of command line tools.
+
+[Sapphire]: ../../build/sapphire/README.mdx
+
+## Prerequisites
+
+Before following this guide, make sure you've followed the [Prerequisites](prerequisites),
+[Run a Non-validator Node](non-validator-node.mdx), and [Run a ParaTime client Node](paratime-client-node.mdx)
+sections and have a working ParaTime client node with [TEE support].
+
+### Stake Requirements
+
+To be able to register as a ParaTime observer node on the Oasis Network, you need to have enough tokens staked in your
+entity's escrow account.
+
+Current minimum staking requirements for a specific ParaTime are listed on the [Stake Requirements] page. You can also
+use [Oasis CLI tools] to check that as described in [Common Staking Info].
+
+Finally, to stake the tokens and to check if you staked correctly, you can use any wallet and any explorer. If using
+our [Oasis CLI tools] and if everything was set up correctly, you should see something like below when running
+[`oasis account show`] command for your entity's account (this is an example for Testnet):
+
+![code shell](../../../external/cli/examples/account/show-delegations.in.static)
+
+![code](../../../external/cli/examples/account/show-delegations.out.static)
+
+:::caution
+
+The stake requirements may differ from ParaTime to ParaTime and are subject to change in the future.
+
+Currently, for example, if you want to register an observer node for Testnet/Mainnet, you currently need to have at
+least **200 TEST/ROSE** tokens delegated:
+- **100 TEST/ROSE** for registering a new node entity and,
+- **100 TEST/ROSE** for gaining an observer role.
```

Comment:
```suggestion
- **100 TEST/ROSE** for each observer node.
```
To make it obvious that for each node you need 100 TEST. We should also change the wording on https://docs.oasis.io/node/run-your-node/prerequisites/stake-requirements/#stake-requirements-by-a-specific-role-of-the-node to make it obvious you need so much for each node.

Reply:
Updated.

For the _Stake Requirements_ I think it it is already clear since it says that _Registration of node_ in singular.